valhalla/baldr/graphconstants.h

```diff
@@ -105,14 +105,15 @@ constexpr uint32_t kMaxAssumedTruckSpeed = 120; // ~75 MPH
 // to measure a probe going this slow via stop and go traffic over a long enough
 // stretch, its unlikely to be good signal below this value
 constexpr uint32_t kMinSpeedKph = 5; // ~3 MPH
+constexpr uint32_t kMinValidSpeedKph = 1;
 
 // Default Fixed Speed. This is the default fixed speed that is assumed.
 // Unless otherwised specified no fixed speed will be assumed and speed will be
 // calculated from costing algorithm.
 constexpr uint32_t kDisableFixedSpeed = 0; // ~0 MPH
 
-inline bool valid_speed(float speed) {
-  return speed > kMinSpeedKph && speed < kMaxAssumedSpeed;
```

Comment:
hm, I don't think we should get rid of the `kMaxAssumedSpeed` validity check. we might run into trouble on the a* heuristic, which is defaulted to that value, so we could end up having a real cost lower than the a* estimate. also, as you mentioned, there don't seem to be higher values than that at least in the europe tomtom data.

Reply:
Good point, but don't we clamp speeds to top_speed, which is what actually informs the heuristic? In that case this check would be redundant.